src/app/src/components/Navbar.jsx

```diff
@@ -218,7 +218,11 @@ function Navbar({ openGDPR }) {
                             href="/"
                             style={{ display: 'inline-flex' }}
                         >
-                            <img src={logo} className="App-logo" alt="logo" />
+                            <img
+                                src={logo}
+                                style={{ height: '60px' }}
```

Comment:
The new image has white space at the top and bottom, causing a 40px high image to appear smaller. Increasing the size corrects this. (Adjusting the CSS didn't correct the issue, despite clearing the cache/etc., but using `styles` directly worked.)